azext_iot/constants.py

```diff
@@ -48,3 +48,7 @@
 
 # Config Key's
 CONFIG_KEY_UAMQP_EXT_VERSION = "uamqp_ext_version"
+
+#IoTC Versions
+PREVIEW = "preview"
```

Comment:
These central API attribute names should either be prefixed as such or we organize differently for example create a new central dictionary in constants that holds all central constant attributes. This way its a lot easier to understand what attributes belong to what service.